<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChannelConfigWidget</class>
 <widget class="QWidget" name="ChannelConfigWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Mode</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="OpTopic_">
           <property name="text">
            <string>Only channel  operator can change topic</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="BlockOutMessage_">
           <property name="text">
            <string>Block messages out of channel </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="SecretChannel_">
           <property name="text">
            <string>Secret channel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="InvitesOnly_">
           <property name="text">
            <string>Only by invire</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="ModerateChannel_">
           <property name="text">
            <string>Moderate channel</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="UserLimit_">
             <property name="text">
              <string>Max users</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="spinBox"/>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QCheckBox" name="Password_">
             <property name="text">
              <string>Password</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>147</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Ban list</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Search_"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTableWidget" name="BanList_"/>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Host mask:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="HostMask_"/>
         </item>
         <item>
          <widget class="QPushButton" name="UpdateBan_">
           <property name="text">
            <string>u</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="AddBan_">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="RemoveBan_">
           <property name="text">
            <string>-</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
